install/kubernetes/ingress-sni-zvpn.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: envoy-config
+  namespace: istio-system
+data:
+  envoy.yaml: |
+    stats_config:
+      use_all_default_tags: false
+
+    admin:
+      access_log_path: /dev/stdout
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 15000 }
+
+    dynamic_resources:
+      lds_config:
+        api_config_source:
+          api_type: REST_LEGACY
+          refresh_delay: 10s
+          cluster_name: [rds]
+      cds_config:
+        api_config_source:
+          api_type: REST_LEGACY
+          refresh_delay: 10s
+          cluster_name: [rds]
+      # Required - without it cds_config doesn't work
+      deprecated_v1:
+        sds_config:
+          api_config_source:
+                api_type: REST_LEGACY
+                refresh_delay: 10s
+                cluster_name: [rds]
+
+    static_resources:
+      listeners:
+      # HTTP - manually configured. Pilot doesn't need to override this, can be customized by user.
+      - name: http80
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 8080 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            # TODO: add mixer
+            config:
+              stat_prefix: http80
+              codec_type: AUTO
+              rds:
+                # Will be passed as part of the URL /v1/routes/80/istio-proxy/ingress~~...
+                route_config_name: "80"
+                config_source:
+                  api_config_source:
+                      api_type: REST_LEGACY
+                      refresh_delay: 10s
+                      cluster_name: [rds]
+                # envoy.router factory returned nullptr instead of empty config message.
+                #              http_filters:
+                #              - name: envoy.router
+                #                deprecated_v1: true
+
+      # HTTPS - currently manually configured, until pilot generates LDSv2 for ingress.
+      # If using kube-lego, certificate can be retrieved for testing with:
+      #   kubectl get secret -n istio-system httpbin-tls -o jsonpath='{.data.tls\.key}' |base64 -d > /etc/istio/httpbin-certs/tls.key
+      #   kubectl get secret -n istio-system httpbin-tls -o jsonpath='{.data.crt\.key}' |base64 -d > /etc/istio/httpbin-certs/crt.key
+      - name: http443
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 8443 }
```

Comment:
wrong port 8443

Reply:
Currently it's setup to have both LDS v1 (from pilot ) and those v2 listeners - which don't work yet completely. If I use port 443 in the manual one - it'll be replace by the LDSv1. 

This is temporary, to test and compare both. Once I get v2 working ( there are at least 2 problems with Oct Envoy that gets into proxy ) - I'll start working on generating the working format from pilot.